api/server/cluster.go

```diff
@@ -426,6 +436,25 @@ func (c *clusterApi) versions(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(versions)
 }
 
+// swagger:operation GET /cluster/alerts/{resource} cluster alerts enumerate enumerateAlerts
+//
+// This will return a list of alerts for the requested resource
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - name: resource
+//   in: path
+//   description: resourcetype to get alerts with
+//   required: true
+//   schema:
+//    "$ref": "#/definitions/ResourceType"
+// responses:
+//   '200':
+//      description: Alerts object
+//      schema:
```

Comment:
type: array

Reply:
Alerts is an object that has an array in it.